skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -66,53 +68,99 @@ export default class search extends Component {
   }
 
   componentDidMount() {
-    fetch("http://localhost:63474/api/Competition/", {
-      method: 'get',
+    console.log("in function");
```

Comment:
Console logs and commited localhost URLS please also make use of a constant for base url